ncm-download/src/main/pan/components/download/config.pan

```diff
@@ -33,4 +33,4 @@ prefix '/software/components/${project.artifactId}';
 
 'active' ?= true;
 'dispatch' ?= true;
-
+'head_timeout' ?= 3;
```

Comment:
Since it's just the HEAD request to check if a file has changed, it should be very fast, so for us 3 seconds is a long time. But it's a good point, it may not be a good default for others. So I think it's best we don't set any default on this, have changed it.